plugins/modules/dcnm_vrf.py

```diff
@@ -335,16 +335,7 @@
     state: query
     config:
     - vrf_name: ansible-vrf-r1
-      vrf_id: 9008011
```

Comment:
Just double checking.  I know that this PR enabled a query using just vrf or network but does it also support other parameters as additional filters?

Are we still testing other filters?